src/tabs/tab-header-bar.tsx

```diff
@@ -223,20 +255,37 @@ export function TabHeaderBar({
         ref={navigationAPI}
         navigationActive={true}
         getNextFocusTarget={getNextFocusTarget}
+        onUnregisterFocusable={onUnregisterFocusable}
       >
-        <ul
-          role="tablist"
-          className={styles['tabs-header-list']}
-          aria-label={ariaLabel}
-          aria-labelledby={ariaLabelledby}
-          ref={headerBarRef}
-          onScroll={onScroll}
-          onKeyDown={onKeyDown}
-          onFocus={onFocus}
-          onBlur={onBlur}
-        >
-          {tabs.map(renderTabHeader)}
-        </ul>
+        {hasActionOrDismissible ? (
+          <div
+            {...tabActionAttributes}
```

Comment:
can we reuse more properties between actionable and non-actionable tabs?

Reply:
For sure! I'll look into how we can make this cleaner